.github/workflows/test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     steps:
       - uses: actions/setup-node@v1
         with:
-          node-version: 12.x
+          node-version: 16.x
```

Comment:
are there any cases where we will fallback to node12? Wondering whether worth having a test matrix

Reply:
Not at this moment, the version defined in the `action.yaml` is the version we use. Once we release a new major version, I can update this and the readme to include more testing (we can cut a releases/v2 branch to mirror our v1 branch) and test both that and the main branch